src/component/index.js

```diff
@@ -0,0 +1,7 @@
+export { CgPlayList } from "react-icons/cg";
+export { BiSearch  } from "react-icons/bi";
+export {BsClockHistory} from "react-icons/bs"
+export {AiOutlineUser,AiOutlineLike} from 'react-icons/ai'
```

Comment:
- you can provide gaps for these also for consistency or use `Prettier` for formatting.

Reply:
okay.